### ISSUE TITLE
Validation on different dataset

### ISSUE THREAD

**Issue by P260** [snap-stanford/pretrain-gnns#34]
Hi, 
I am using your model to use pretrained model and fine-tuning on my own dataset. Is it possible to validate my fine-tuned model on another different validation dataset? I want to keep it completely separate. Is this possible with pretrain-gnns chem model? If yes, how can I approach?  

Thanks.

**Comment by weihua916** [comment]
Yes, I think the most relevant code is this: https://github.com/snap-stanford/pretrain-gnns/blob/master/chem/finetune.py

It reads the pre-trained model, finetunes it, and evaluates it.

**Comment by P260** [comment]
Hi @weihua916 ,  yes I understood that. My concern is I want to check 100% as validation set, instead of splitting in train, test, valid. Is that possible with finetune.py script?
